src/@next/components/atoms/Attribute/types.ts

```diff
@@ -7,4 +7,8 @@ export interface IProps {
    * Value
    */
   attributeValue: string;
+  /**
+   * Data test for value
+   */
+  valueDataTest?: string;
```

Comment:
In repository we are using different name for test selectors. Button component for reference https://github.com/mirumee/saleor-storefront/blob/master/src/%40next/components/atoms/Button/types.ts#L24